shared/css/homepage_below_hero_plane_banner.scss

```diff
@@ -20,7 +20,7 @@ $blue_background_colour: #59b9dc;
   padding-bottom: 5px;
 
   a {
-    font-family: 'Gotham 4r', sans-serif;
+    font-family: "Gotham 4r", sans-serif;
```

Comment:
Just curious -- does this change cause a functional difference?

Reply:
Nope! This was just a result of "format on save" being set in my editor, so maybe there's a linting rule that wasn't being adhered to here.